chaos/litmus/master.yaml

```diff
@@ -0,0 +1,66 @@
+---
+# mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
+# fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
+# Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
+# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
+# will help with categorization & application-mapping
+
+
+litmus_test_1:
+  name: pod_kill
+  test_tags: ['mysql', 'pumba']
+  location: https://raw.githubusercontent.com/openebs/litmus/master/apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
+  test_params:
+    - name: DB_SERVER_IP
+      kind: env
```

Comment:
what may be other kinds?

Reply:
env & configmaps are the only defined kinds in the current litmus tests.